src/js/components/sharedComponents/TreemapCell.jsx

```diff
@@ -60,7 +60,7 @@ export default class TreemapCell extends React.Component {
     }
 
     clickedCell() {
-        if (this.props.data.id) {
+        if (this.props.data.id !== 'undefined') {
```

Comment:
```suggestion
        if (this.props.data.id &&  this.props.data.id !== 'undefined') {
```
I think we still want to handle the case where `id` is another falsy value. I'm also wondering if there's a better way to handle this upstream before the value `undefined` is getting cast to a string somewhere